pkg/cli/cmds/nocluster.go

```diff
@@ -1,4 +1,4 @@
-// +build !dqlite
+// +build no_etcd
 
 package cmds
 
```

Comment:
I'm guessing this failed the case-sensitive search through the code for "dqlite":
```
const (
	hideDqlite = true
)
```

Reply:
Thinko'd right over the const name. Thanks.